packages/api/src/command/medical/patient/__tests__/append-doc-query-progress.test.ts

```diff
@@ -324,234 +322,3 @@ describe("appendDocQueryProgress", () => {
     expect(patientSentToUpdate?.data === patientModel.data).toBeFalsy();
   });
 });
-
```

Comment:
Remove these tests because instead of testing the entire `setDocQueryProgress` im testing the 2 main function in it `setHIEDocProgress`  & `aggregateAndSetHIEProgresses`